Source/Core/Color.js

```diff
@@ -302,11 +299,9 @@ define([
             var maximumGreen = defaultValue(options.maximumGreen, 1.0);
 
             //>>includeStart('debug', pragmas.debug);
-            if (minimumGreen > maximumGreen) {
-                throw new DeveloperError("minimumGreen must be less than or equal to maximumGreen");
-            }
+            
             //>>includeEnd('debug');
-
+            Check.typeOf.number.lessThanOrEquals('minimumGreen', minimumGreen, maximumGreen);
```

Comment:
Throughout, make sure the `Check` call is between `//>>includeStart('debug', pragmas.debug);` and `//>>includeEnd('debug');`. The purpose of those is to mark code that should not be included in the release build.